pkg/issuer/vault/issue.go

```diff
@@ -214,17 +215,19 @@ func (v *Vault) requestTokenWithAppRoleRef(client *vault.Client, appRole *v1alph
 	return token, nil
 }
 
-func (v *Vault) requestVaultCert(commonName string, certDuration time.Duration, altNames []string, csr []byte) ([]byte, []byte, error) {
+func (v *Vault) requestVaultCert(commonName string, certDuration time.Duration, altNames []string, ipSans []string, csr []byte) ([]byte, []byte, error) {
```

Comment:
I think we may be getting to the point of needing a `vaultCertRequest` structure to hold all of the potential options that could be set here, or otherwise refactoring this function in some way to accept a CSR + whatever additional fields are required. This doesn't need to be done as part of this PR though 😄

Reply:
So if you agree, I close this point